syntax = "proto3";
import "google/protobuf/descriptor.proto";

extend google.protobuf.MessageOptions {
	string primary_key = 50000;
}

extend google.protobuf.FieldOptions {
	uint32 serialize_type = 50001;
}
